### ISSUE TITLE
Always same problem, miners didnt starting on some p

### ISSUE THREAD

**Issue by error0x1337** [UnamSanctam/SilentCryptoMiner#1949]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/150927220/cdc89a43-f126-4278-b2e4-c1819b0d6ed3)
Miner is not starting, same thing happened 2 weeks ago to many PCs but one of them appeared later on the panel, What could be the problem of this, what should cause this ? I Bypass every antivirus with my loader, + your miner is also UD to most antivirus (3.4.0 version)

**Comment by UnamSanctam** [comment]
Hmm that's weird, are you sure that your loader is loading it correctly? It could be that something goes wrong on some computers. You can also try setting the "Inject Into" option to explorer.exe if you're not already using that, for me conhost.exe worked better but in one issue a person received some memory detections with conhost.exe

**Comment by error0x1337** [comment]
Ok, ill add the feature that will check if the miner is installed or not.
If the miner is installed it should create that file right? 
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/150927220/1dc27c81-7127-42c8-ad0e-a33de339b25a)

**Comment by UnamSanctam** [comment]
Yes exactly, if it's installed as an administrator then it should (usually) be inside C:\ProgramData\ (%ProgramData%) and then the "File Name" in your screenshot.

**Comment by error0x1337** [comment]
Alright, great

**Comment by error0x1337** [comment]
But i cant build the miner with administrator privilegies on 3.4.0 version

**Comment by UnamSanctam** [comment]
In the new version it will ask for administrator privileges (if it does not already have that) using the code instead of a manifest file. That's because when you have a manifest file that requires administrator privileges then antiviruses find that really suspicious, so asking for them programmatically is much less detected.

**Comment by error0x1337** [comment]
Oh, then yeah im running it as an administrator, I didnt checked it before lol.. cus im also using administrator manifest on my loader and testing miner there

**Comment by error0x1337** [comment]
Bro @UnamSanctam I cant find the entry name in the task scheduler on my VM, after installed the miner successfully
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/150927220/af7e83d5-1fbd-4bb7-96bf-05f21872eb76)

**Comment by UnamSanctam** [comment]
It now gets installed as a service, so check your services.

**Comment by error0x1337** [comment]
Oh

**Comment by error0x1337** [comment]
I found thanks

**Comment by error0x1337** [comment]
How to remove it xd

**Comment by UnamSanctam** [comment]
It's best to run the uninstaller, otherwise you can open an administrator CMD and run `sc delete "ENTRYNAME" ` though if you use the watchdog then you'll somehow have to kill that first.

**Comment by error0x1337** [comment]
Yes im using watchdog

**Comment by error0x1337** [comment]
But how to kill it in task manager

**Comment by error0x1337** [comment]
I just deleted service

**Comment by UnamSanctam** [comment]
>But how to kill it in task manager

You'll have to find the conhost.exe that the watchdog is injected into (you'll need to guess since it will look just like any others) and kill that. You'll probably have to use something like Process Hacker to be able to find it easier. Though I would recommend using the uninstaller instead if you still have it for that miner build.

>I just deleted service, It will not work?

If the watchdog is running then it should add the service back again, it checks every 5 seconds.

**Comment by Mustwey** [comment]
> ![image](https://user-images.githubusercontent.com/150927220/284071460-cdc89a43-f126-4278-b2e4-c1819b0d6ed3.png) Miner is not starting, same thing happened 2 weeks ago to many PCs but one of them appeared later on the panel, What could be the problem of this, what should cause this ? I Bypass every antivirus with my loader, + your miner is also UD to most antivirus (3.4.0 version)

what loader are u using?
